float64 delta_x
float64 delta_y
float64 delta_z
---
float64 estimated_final_x
float64 estimated_final_y
float64 estimated_final_z
---
